pallets/msa/src/runtime-api/src/lib.rs

```diff
@@ -31,9 +31,9 @@ sp_api::decl_runtime_apis! {
 		// fn get_msa_keys(msa_id: MessageSourceId) ->	Vec<KeyInfoResponse<AccountId>>;
 
 		/// Check to see if a delegation existed between the given delegator and provider at a given block
-		fn has_delegation(delegator: Delegator, provider: Provider, block_number: Option<BlockNumber>) -> bool;
+		fn has_delegation(delegator: DelegatorId, provider: ProviderId, block_number: Option<BlockNumber>) -> bool;
 
 		/// Get the list of schema ids (if any) that exist in any delegation between the delegator and provider
-		fn get_granted_schemas_by_msa_id(delegator: Delegator, provider: Provider) -> Option<Vec<SchemaId>>;
+		fn get_granted_schemas_by_msa_id(delegator: DelegatorId, provider: ProviderId) -> Option<Vec<SchemaId>>;
```

Comment:
Runtime APIs are also not automatic, and requires an update to https://github.com/LibertyDSNP/frequency/blob/main/js/api-augment/definitions/msa.ts